MAUI/Kanban-Board/Column.md

```diff
@@ -277,3 +277,353 @@ kanban.Columns.Add(todoColumn);
 {% endhighlight %}
 
 {% endtabs %}
+
+## Appearance Customization
+
+### Customizing Backgrounds
```

Comment:
Customize the column background

Use the `Background` property of the `KanbanColumn` to customize the background color of each column in the Kanban control.